### ISSUE TITLE
can't use own tempaltes with pdoc

### ISSUE THREAD

**Issue by undertherain** [mitmproxy/pdoc#178]
I'm trying to use my own templates with pdoc,
I'm specifying path to templates (for starters I took existing templates from this repo and modified them slightly)
I specify `--template-dir` option, but still see the result rendered with default templates

is there any way to, first of all,  see if it is trying even to use new templates? If I provide some garbage as template path it still works 

I'm using pdoc 0.3.2 on Python 3.6.5, Ubuntu 18.04.1

**Comment by undertherain** [comment]
Even interestingly, that was happening if I use --html option
if I use default text mode, then if I add template path, I'm getting the following error 
```
Traceback (most recent call last):
  File "/home/blackbird/.local/bin/pdoc", line 483, in <module>
    output = module.text()
  File "/home/blackbird/.local/lib/python3.6/site-packages/pdoc/__init__.py", line 652, in text
    text, _ = re.subn('\n\n\n+', '\n\n', t.render(module=self).strip())
  File "/home/blackbird/.local/lib/python3.6/site-packages/mako/template.py", line 462, in render
    return runtime._render(self, self.callable_, args, data)
  File "/home/blackbird/.local/lib/python3.6/site-packages/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/home/blackbird/.local/lib/python3.6/site-packages/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/home/blackbird/.local/lib/python3.6/site-packages/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "_text_mako", line 86, in render_body
TypeError: object of type 'method' has no len()
```

**Comment by undertherain** [comment]
using mako version '1.0.7'

**Comment by KyleKing** [comment]
I have the same issue with the latest pdoc (`0.3.2`) on Windows 7 and Python 3.7.2
I copied the templates from the pdoc templates folder, then tried different modifications to the `css.mako.txt` and `html_index.mako.txt` file, but pdoc won't load the templates regardless if from `~/.config/pdoc`, `~/.config/pdoc/templates`, or specifying a full or relative path with `--template-dir` (e.g. for the relative directory: `pdoc image_utils --html --html-dir docs --overwrite  --template-dir templates`)

**Comment by KyleKing** [comment]
In case this helps with troubleshooting this issue:

I was using Anaconda on Windows with a conda-venv for Python 3.7.2 (base was 2.7.15) {So I thought that might be the issue}.
I did a fresh install of Python 3.6.7 from [python.org/](https://www.python.org/downloads/release/python-367/) and modified my PATH to remove Anaconda. I needed to run pdoc with the absolute path based on https://github.com/mitmproxy/pdoc/issues/127#issuecomment-294449224. I tried variations of what I thought should work:

```sh
$ which python
/c/Users/king.kyle/Python36/python
$ python --version
Python 3.6.7
$ pip install pdoc
$ python C:/Users/king.kyle/Python36/Scripts/pdoc csv --html --overwrite --template-dir templates
$ python C:/Users/king.kyle/Python36/Scripts/pdoc csv --html --overwrite --template-dir C:/Users/king.kyle/templates/
$ python C:/Users/king.kyle/Python36/Scripts/pdoc csv --html --overwrite --template-dir templates/html_frame.mako.txt
```

The CSS file is modified, so there should be some visible difference if the custom templates were loaded:

![diff](https://user-images.githubusercontent.com/3784339/53269563-348bc100-36b7-11e9-9eb3-ec5b909c0161.png)

Any thoughts on what might be wrong? I'll test this on Mac later today. Maybe this is just Windows only. Is there any verbose/debugging that can be enabled?
